src/styles/_variables.scss

```diff
@@ -0,0 +1,44 @@
+//  --------------------------------------------------------------------------------
+//  Contains all application-wide Sass variables
+//  --------------------------------------------------------------------------------
+
+$url-background-light-image: "../assets/bg-light2.jpg";
+$url-background-dark-image: "../assets/bg-dark1.jpg";
+
+$color-yellow-base: rgb(255, 167, 0);
+$color-yellow-active: rgb(253, 153, 2);
+$color-brown-base: rgb(49, 11, 8);
+$color-brown-active: rgb(41, 7, 6);
+$color-yellow-base-theme: 255, 167, 0;
+$color-yellow-active-theme: 253, 153, 2;
+$color-brown-base-theme: 49, 11, 8;
+$color-brown-active-theme: 41, 7, 6;
+$color-gray-dark: gray;
```

Comment:
Its better to use css variables specialy if you want to use dark/light theme

Reply:
Why? 
I read this on topic of sass based projects&dark mode and used it on my previous project: https://dev.to/zetareticoli/dark-mode-with-sass-and-css-variables-4f9b

Why not:
1)
 have partial scss file(s) with sass variables declared (if you declare them as css variables, they need to be declared under :root and not being a pure selector it needs to be in globals.scss)? 
2)
@use partial scss file with sass variables in globals.scss and under :root:not([dark]) and :root([dark]) store as css variables only those sass variables that are needed for dark/light mode?
3)
call css variables when styling components/attributes that change their style depending on dark/light mode

PROS:
- cleaner organisation 
- can immediately see in each stylesheet if/which attribute changes with dark/light mode or not (because only attributes that change call css variables, all others call sass variables)

CONS:
- need to be careful to use proper escaping Sass syntax: --variable: #{$variable}; 
even more so with rgba: attribute: #{"rgba(var(--variable), opacity)"};
- using different names for the same property value can sometimes be inconvenient

Reply:
Yea ok. If you can make it work go for it